docs/content/developer/tutorials/identity-workshop.mdx

```diff
@@ -0,0 +1,1417 @@
+
+### What You’ll Build in this Workshop
+
+We're building a complete, system that connects a modern React frontend to a high-performance Rust backend. You'll learn how the frontend triggers the entire IOTA Identity lifecycle:
+
+- **Student (Holder)**: Creates a **Decentralized Identifier (DID)** and stores it securely.
+- **University (Issuer)**: Issues a **Verifiable Credential (VC)** representing the degree.
+- **Student**: Packages the credential into a **Verifiable Presentation (VP)** and signs it.
+- **Employer (Verifier)**: Verifies the authenticity of the presentation and credentials against the IOTA network.
+
+## Alternative Use Cases for IOTA Identity
+
+### Use Case 2: Our Digital Lives are Fragmented and Vulnerable
+
+Your online identity is scattered across platforms, leaving it vulnerable to misuse and attack.
+
+We hold dozens of accounts (social media, banking, email), each requiring a separate login and sharing pieces of our personal data. This fragmentation makes managing security a nightmare, often leading to recycled passwords and creating massive centralized targets for breaches. When a company database is breached, your information is exposed, and you have zero control over it. IOTA Identity solves this by giving you a single, Decentralized Identifier (DID) that you control, allowing you to selectively prove facts about yourself without exposing the underlying data, putting you back in charge.
+
+### Use Case 3: Consumers Can’t Trust a Product’s History
+
+Is that product authentic? Where did it really come from? Should I believe its green claims?
+
+The market is flooded with counterfeit goods, causing financial loss and safety hazards (like fake medications). For legitimate products, the journey from raw materials to retail is often opaque. Consumers demand transparency regarding ethical sourcing, environmental impact, and authenticity. Without a reliable way to track a product's history and verify the identity of every supplier and auditor involved, trust is impossible. IOTA Identity enables the creation of verifiable, tamper-proof credentials at each point in the supply chain, establishing a trusted, auditable history for any product.
+
+### Use Case 4: Verifying Digital Asset Provenance
+
+Link digital assets and their owners with verifiable, tamper-proof identities.
+
+The digital asset space, from NFTs to tokenized real estate, faces a critical challenge: proving an asset's authenticity and origin. How can a buyer be sure a digital token was created by the legitimate originator, or that a tokenized asset is verifiably linked to its physical counterpart and owner? Without a robust, cryptographically secured link between the digital token and the real-world identity and provenance, the system is vulnerable to fraud and impersonation. IOTA Identity provides the essential cryptographic layer necessary to secure these digital-to-physical relationships and drive mainstream adoption.
+
+## Prerequisites
```

Comment:
Prerequisites
   Installing Rust and Cargo
   Run Local IOTA Network
   Install VS Code